### DISCUSSION TITLE
Implement Remote Shell/Terminal capabilities similar to N-Central Take Control

### DISCUSSION THREAD

**Discussion by Reinitialized** [rustdesk/rustdesk#1175] [General]
As it currently stands, I see no way to initate a remote shell within RustDesk. This would bring feature parity to this already amazing foss project when compared to already existing solutions.

A rundown of exactly what I have envisioned:
- Integrated Remote Shell functionality similar to that of N-Central Take Control: 
![image](https://user-images.githubusercontent.com/20674500/182669662-1b576980-323f-4601-8d7a-140dfd7de4d7.png)
- Terminal should be similar in functionality to TacticalRMM Remote Background in which you can paste into the terminal and respond to keyboard prompts

**Comment by Barna13** [comment]
I was also a Nable customer, and I miss the remote background feature, even though this feature only needs to be implemented in Pro plans for security reasons.
This feature also displays the host's operating status, e.g. RAM Usage/CPU, SMART information, etc.
